x/ecocredit/simulation/genesis.go

```diff
@@ -225,32 +268,47 @@ func genGenesisState(ctx context.Context, r *rand.Rand, simState *module.Simulat
 
 	pKey2, err := ss.ProjectInfoTable().InsertReturningID(ctx,
 		&api.ProjectInfo{
-			Key:                 cKey2,
-			Id:                  "P02",
-			Admin:               accs[1].Address.Bytes(),
+			ClassKey:            cKey2,
+			Admin:               accs[1].Address,
 			ProjectJurisdiction: "AQ",
 			Metadata:            metadata,
+			Id:                  "C0201",
```

Comment:
Ok to leave as is but the project id might not include the credit class id #1046